packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth.ts

```diff
@@ -2261,6 +2296,56 @@ describe("Eth module", function () {
             0
           );
         });
+
+        it("should return the right info for the existing transaction in the context of a new block with 'pending' block tag param", async function () {
```

Comment:
This seems too long, maybe change to:
```suggestion
        it("should return the right transaction info when called with 'pending' block tag param", async function () {
```